singularity/code/graphics/button.py

```diff
@@ -121,19 +158,17 @@ def _new_hotkey(self, value):
         super(Button, self)._new_hotkey(value)
 
     def __init__(self, parent, pos, size = (0, .045), base_font = None,
-                 borders = constants.ALL, text_shrink_factor = .825,
-                 priority = 100, **kwargs):
+                 borders = constants.ALL, text_shrink_factor = .825, **kwargs):
         self.parent = parent
 
-        self.priority = priority
-
         kwargs.setdefault('text_size', 'button')
         super(Button, self).__init__(parent, pos, size, **kwargs)
 
         self.base_font = base_font or "special"
         self.borders = borders
         self.shrink_factor = text_shrink_factor
 
+        self.hotkey_func = lambda event: self.activate_with_sound(event)
```

Comment:
```suggestion
        self.hotkey_func = self.activate_with_sound
```

(Same thing; just without a lambda indirection)

Reply:
Of course